test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/ServerTimingHeaderTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// <copyright file="ServerTimingHeaderTests.cs" company="Splunk Inc.">
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET6_0_OR_GREATER
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests;
+
+public class ServerTimingHeaderTests : TestHelper
+{
+    public ServerTimingHeaderTests(ITestOutputHelper output)
+        : base("HttpServer", output)
+    {
+    }
+
+    [Fact]
+    public async Task SubmitRequest()
```

Comment:
What about verify if there is no header when the env. is disabled - mainly to proof that env. variable is taken into account.

Reply:
https://github.com/signalfx/splunk-otel-dotnet/pull/94/commits/1fd6cec02887546af7b5e966f472a34ad2950be7